src/Builder/Visitor/RemovePluginFromPluginListVisitor.php

```diff
@@ -93,6 +93,11 @@ public function filterStatements(Node $node): Node
         foreach ($node->stmts as $index => $stmt) {
             if ($stmt instanceof If_) {
                 $stmt = $this->filterStatements($stmt);
+                if (!$stmt->stmts) {
```

Comment:
I guess this check should be before `filterStatements` call because because anyway we remove this node

Reply:
The place is correct. changes come after function closure